project.mml

```diff
@@ -1483,6 +1484,8 @@ Layer:
                 'natural_' || CASE WHEN "natural" IN ('wood', 'water', 'mud', 'wetland', 'bay', 'spring', 'scree', 'shingle', 'bare_rock', 'sand', 'heath',
                                                       'grassland', 'scrub', 'beach', 'glacier', 'tree', 'strait', 'cape')
                                                       THEN "natural" END,
+                -- mountain_pass come after the natural=saddle, to give priority to nodes being both a mountain_pass and a saddle
```

Comment:
"comes", and remove "the" before "natural=saddle". Or maybe this could be shorted into a comment on the next line like `  // after natural=saddle to give priority to that tag on the same node`